specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/preview/2020-01-15-preview3/CommunicationIdentity.json

```diff
@@ -0,0 +1,261 @@
+      "post": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Create a new identity.",
+        "operationId": "CommunicationIdentity_Create",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns the created identity.",
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentity"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create Identity": {
+            "$ref": "./examples/CreateIdentity.json"
+          }
+        }
+      }
+    },
+    "/identities/{id}": {
+      "delete": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Delete the identity, revoke all tokens of the identity and delete all associated data.",
+        "operationId": "CommunicationIdentity_Delete",
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Identifier of the identity to be deleted.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Delete Identity": {
+            "$ref": "./examples/DeleteIdentity.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "Identity"
+        ],
+        "summary": "Update an Identity.",
+        "operationId": "CommunicationIdentity_Update",
+        "consumes": [
+          "application/merge-patch+json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "id",
+            "description": "Identifier of the identity.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "api-version",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The properties of the identity to be updated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CommunicationIdentityUpdateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Success"
+          }
+        },
+        "x-ms-examples": {
+          "Update Identity": {
+            "$ref": "./examples/UpdateIdentity.json"
+          }
+        }
+      }
+    },
+    "/identities/{id}/~issue-token": {
```

Comment:
According to guidelines I think we should use camelCase, so `~issueToken`

Reply:
Let me introduce another sources. From this page : https://restfulapi.net/resource-naming/, you can see "Use hyphens (-) " and also "use lower case" as standard, action-name  for controller resource was a result of preview API review, and ~action-name is some new standard created recently. So IMHO these resources of standards should be unified. Let's discuss this on teams.

Reply:
Updated